features/font-display.yml

```
@@ -0,0 +1,7 @@
+name: font-display
+description: The `font-display` CSS descriptor sets what is displayed while a font face loads.
```

```suggestion
description: The `font-display` CSS descriptor sets whether to show a substitute font or nothing while a font face loads. The property manages what's known as a flash of unstyled text or flash of invisible text.
```

What do you think of an elaboration like this? The first bit is to avoid a circular description ("font-display … sets what is displayed"), while the second is address what this is for, though maybe it's being a bit too assertive.

---

I like that, and adding that specific use case makes sense.